scripts/gWMS-CMSRunAnalysis.sh

```diff
@@ -61,23 +154,25 @@ function chirp_exit_code {
         echo "Cannot find condor_chirp to set up the job exit code (ignoring, that's for monitoring purposes)"
         return
     fi
+
     #check if exitCode is in there and report it
-    #any error or exception will be printed to stderr and cause $? <> 0
-    LONG_EXIT_CODE=$(python << END
-from __future__ import print_function
-import json
-with open("jobReport.json.$CRAB_Id") as fd:
-    data = json.load(fd)
-print(data['exitCode'])
-END
-)
-    if [ $? -eq 0 ]; then
-        echo "==== Long exit code of the job is $LONG_EXIT_CODE ===="
-        condor_chirp set_job_attr_delayed Chirp_CRAB3_Job_ExitCode $LONG_EXIT_CODE
-        CHIRP_EC=$?
-    else
-        echo "==== Failed to load the long exit code from jobReport.json.$CRAB_Id. Falling back to short exit code ===="
-        #otherwise checjk if EXIT_STATUS is set and report it
+    #otherwise check if EXIT_STATUS is set and report it
+    if [[ $EXITTXT_NONEMPTY ]]; then
+        #any error or exception will be printed to stderr and cause $? <> 0
```

Comment:
do you expect error or exceptions from `cat` ?

Reply:
mmhh, cat returns an error only if the file does not exist, and we already checking for it. We can be a little bit less defensive and assume that if `jobReport.exitCode.txt`, then it contains something that we can read.

I simplified a bit the logic that was inherited from parsing `jobReport.json`, thanks!